vega_sim/parameter_test/parameter/loggers.py

```diff
@@ -62,6 +66,7 @@ def ideal_market_maker_single_data_extraction(
 def _ideal_market_maker_single_data_extraction(
     vega: VegaServiceNull,
     agents: List[Agent],
+    state_values: List = None,
```

Comment:
Still not really sure I like this vs just calculating derived values at a later time, though in the interests of expediency we can probably stick with it for now